src/middlewares/createMetadataMiddleware.ts

```diff
@@ -50,6 +51,7 @@ function extractPayload(
 
     payload.widgets.push({
       type: widget.$$type,
+      widgetType: widget.$$widgetType,
```

Comment:
If I understand correctly `widget.$$type` is the connector type and `widget.$$widgetType` is the actual widget type, is that right ?

If that the case maybe it would be clearer to rename `type` to `connectorType` in the payload ?

Reply:
widgets without a connector (index, analytics) technically set type too, which is why I didn't rename it to connectorType :)

Reply:
What if we clean up the types for widgets without a connector? Is it a good time to do so? No strong opinion, but genuinely asking.